src/cmap/connect.ts

```diff
@@ -85,7 +86,7 @@ function performInitialHandshake(
   }
 
   const authContext = new AuthContext(conn, credentials, options);
-  prepareHandshakeDocument(authContext, (err, handshakeDoc) => {
+  prepareHandshakeDocument(conn.serverApi, authContext, (err, handshakeDoc) => {
```

Comment:
nit: why not store the `serverApi` in the `AuthContext`?

Reply:
Err, right, I assumed that the auth context would really only store auth info, but since the connection itself is also stored on it, that’s not even necessary :)

Thank you for pointing this out, fixed (and simplified) :)